### ISSUE TITLE
Using fine tuned SAM model for live segmentation

### ISSUE THREAD

**Issue by haveragoyal** [DreamJarsAI/Apply-AI-like-a-Pro#1]
How to use fine tuned Segment anything model on custom dataset for live camera segmentation? 
As in your notebook [231112_MedSegmentAnything_FineTuning_1epoc.ipynb](https://github.com/DreamJarsAI/Apply-AI-like-a-Pro/blob/6ce9ecb207177025bf046df638a570a57b29f922/AI%20Interest%20Group%20Materials/231112_MedSegmentAnything_FineTuning_1epoc.ipynb), you have demonstrated how to do inference on test dataset. But during live camera feed, we don't have ground truth masks so how we will prepare dataset for inference like this:

```
# create test dataloader
test_dataset = SAMDataset(image_paths=data_paths['test_images'], mask_paths=data_paths['test_masks'], processor=processor)
test_dataloader = DataLoader(test_dataset, batch_size=1, shuffle=False)
```
